metadata.rb

```diff
@@ -4,7 +4,7 @@
 license          'Apache-2.0'
 description      'Installs/Configures Microsoft SQL Server'
 long_description IO.read(File.join(File.dirname(__FILE__), 'README.md'))
-version          '5.3.1'
+version          '5.4.1'
```

Comment:
Can you remove updating metadata from this commit? We generally update the metadata when we release the cookbook.